flocker/node/agents/cinder.py

```diff
@@ -529,7 +530,7 @@ def list_volumes(self):
         http://docs.rackspace.com/cbs/api/v1.0/cbs-devguide/content/GET_getVolumesDetail_v1__tenant_id__volumes_detail_volumes.html
         """
         flocker_volumes = []
-        for cinder_volume in self.cinder_volume_manager.list():
+        for cinder_volume in self.cinder_volume_manager.list(detailed=False):
```

Comment:
This is because mimic doesn't (yet) emulate the `volumes/detail` endpoint. 
I don't think we need the detail anyway, but the build results should tell us.